crates/driver/src/tests/setup/new/mod.rs

```diff
@@ -114,10 +114,22 @@ pub struct Setup {
     pools: Vec<Pool>,
     orders: Vec<Order>,
     trusted: HashSet<&'static str>,
-    bogus_calldata: bool,
     internalize: bool,
     now: infra::time::Now,
     config_file: Option<PathBuf>,
+    solutions: Vec<Solution>,
+}
+
+#[derive(Debug, Clone, Copy)]
+pub enum Solution {
+    /// Set up the solver to return a valid solution.
+    Valid,
+    /// Set up the solver to return a valid solution, with additional random
```

Comment:
nit: The additional data is hardcoded.
```suggestion
    /// Set up the solver to return a valid solution, with additional
```